### PR TITLE
Add main page.

### PR DESCRIPTION
**Pull request by SG-coder**

### CODE REVIEW

File: src/Pages/MainPage.jsx [hunk 1]
```diff
@@ -0,0 +1,233 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { requestSignup } from '../Store/Actions/auth';
+import Icon from '../components/Icon.jsx';
```
Comment (silwalanish) [inline src/Pages/MainPage.jsx:5]: Paths are case-sensitive. 
```suggestion
import Icon from '../Components/Icon';
```

File: src/Pages/Home.jsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import MainPage from './MainPage.jsx';
```
Comment (silwalanish) [inline src/Pages/Home.jsx:2]: Also move this import to line no 5.
```suggestion
import MainPage from './MainPage';
```

File: src/iconSettings.js [hunk 3]
```diff
@@ -22,7 +22,12 @@ import {
...
   faSpinner,
 } from '@fortawesome/free-solid-svg-icons';
-
+import {
+  faFacebook,
+  faTwitter,
+  faInstagram,
+  faLinkedin,
+} from '@fortawesome/free-brands-svg-icons';
 const ICONS = {
```
Comment (silwalanish) [inline src/iconSettings.js:31]: Add a new line
```suggestion

const ICONS = {
```

**Review by silwalanish — CHANGES_REQUESTED** [review]

File: src/Pages/MainPage.jsx (hunk 1 above)
Comment (silwalanish) [reply src/Pages/MainPage.jsx:5]: This should also fix the warning in windows.

File: src/Pages/MainPage.jsx [hunk 4]
```diff
@@ -0,0 +1,233 @@
+import React from 'react';
```
Comment (silwalanish) [inline src/Pages/MainPage.jsx:1]: We don't need this as separate page. Move the content to `Home.jsx`

File: src/Pages/Home.jsx [hunk 5]
```diff
@@ -19,8 +21,214 @@ class Home extends React.Component {
             <p>Welcome {this.props.user && this.props.user.username}</p>
           </div>
         )}
```
Comment (silwalanish) [inline src/Pages/Home.jsx:23]: Remove contents from line number 18 - 21.

**Review by silwalanish — APPROVED** [review]

**Review by danny237 — APPROVED** [review]

**Review by silwalanish — APPROVED** [review]

**Pull request merged**